docker-compose.test.unit.yaml

```diff
@@ -147,10 +147,10 @@ services:
     networks:
       - pcs
   redfish-simulator:
-    image: artifactory.algol60.net/csm-docker/stable/hms-redfish-translation-service:1.13.3
+    image: artifactory.algol60.net/csm-docker/unstable/hms-redfish-translation-service:1.20.0-20220720200402.0d4a07d
```

Comment:
Once https://github.com/Cray-HPE/hms-redfish-translation-layer/pull/41 merged need to update this image tag.